site/src/docs/components/header-universal-bar/code.mdx

```diff
@@ -0,0 +1,51 @@
+---
+slug: '/components/header-universal-bar/code'
+title: 'HeaderUniversalBar - Code'
+---
+
+import { Header, HeaderUniversalBar, NavigationLink } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Code
+
+### Code examples
+
+#### Default
+
+<InternalLink href="/components/navigation-link">Navigation links</InternalLink> can be put inside the header universal component indepedently as sibling elements or inside a container element.
+
+<Playground>
+
+```jsx
+<Header>
+  <HeaderUniversalBar primaryLinkHref="#" primaryLinkText="Helsingin Kaupunki">
```

Comment:
Lower-case initial letter in the word kaupunki: Helsingin Kaupunki --> Helsingin kaupunki